agent/api/transitiondependency.go

```diff
@@ -1,5 +1,12 @@
 package api
 
+import (
```

Comment:
Can you add a header for this file, not sure how we missed it?

Reply:
just noticing this. thanks, will add.